### ISSUE TITLE
new feature :search

### ISSUE THREAD

**Issue by vaniot-s** [jointwt/twtxt#285]
**Is your feature request related to a problem? Please describe.**
Finding it becomes difficult when there are too many twts

**Describe the solution you'd like**

Hope you can add a search box to find what you need

**Comment by prologic** [comment]
Hello @vaniot-s Welcome to the project and thanks for filing this issue!

Can you give some thought to how this would work for you? What would you be searching for? What would make it easier to find what you're looking for?

Remember too that Twt(s) eventually fall off the active cache (_timeline_) and are archived. So a search for searching archived Twt(s) also makes sense IHMO.

Would we just search on #hashtag(s) or full-text search?

**Comment by vaniot-s** [comment]
@prologic I hope it support full-text search,I can  find twts by keywords

**Comment by prologic** [comment]
> @prologic I hope it support full-text search,I can find twts by some keywords

Yeah okay I was afraid you'd say that 😀 No worries, full-text indexing and searching is a bit tricky to do with the goals of keeping pods small and lightweight, but doable.

One option can be to integrate the use of the very nice [Go](https://golang.org) library [bleve](https://github.com/blevesearch/bleve) which would work quite well I think. It _could_ also easily index #hashtag(s).

**Comment by vaniot-s** [comment]
Okay, I look forward to new features, thank you for your contribution

**Comment by prologic** [comment]
If anyone wants to have a go at this, there's a great getting started guide over at http://blevesearch.com/docs/Getting%20Started/

Looks pretty straight forward...

**Comment by prikeshsavla** [comment]
I'll check this out. If its possible for me or not to integrate. I am thinking a search field in the header to start the search and then leading to a new page with a timeline like view of all the feeds found. Sounds good?

**Comment by prologic** [comment]
What do you mean by a search field in the header?￼

**Comment by prikeshsavla** [comment]
Wouldnt you need a form field to search?

**Comment by prologic** [comment]
Oh yes I see what you see
